### PR TITLE
[docs] Update AABB `get_support` description

### PR DESCRIPTION
**Pull request by beyarkay**
This is a simple documentation edit. It updates the description of the `get_support` method in `AABB` to explain what a support point is.

### CODE REVIEW

File: doc/classes/AABB.xml
Comment (AThousandShips) [inline doc/classes/AABB.xml:131]:
```suggestion
				This point is commonly known as the support point in collision detection algorithms.
```
Keep on one line

File: doc/classes/AABB.xml
Comment (aaronfranke) [inline doc/classes/AABB.xml:130]:
```suggestion
				Returns the vertex of the AABB that's the farthest in a given direction. This point is commonly known as the support point in collision detection algorithms.
```
I would go even further and make the entire thing one line.

In the XML docs, every line break is a new paragraph.

Comment (beyarkay) [reply doc/classes/AABB.xml:130]: Ah my apologies, I didn't realize line breaks cause a new paragraph

**Review by clayjohn — COMMENTED** [review]
Looks great! The final step before merging is to squash all the commits together so that the whole PR only contains 1 big commit with all your changes. We like to merge one commit at a time to keep the git history clean and navigable. 

If you don't know how to do that, we have a helpful tutorial in the official documentation https://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html#the-interactive-rebase

**Comment by beyarkay** [comment]
Apologies for the delay, I've squashed all commits

**Review by aaronfranke — APPROVED** [review]

**Comment by beyarkay** [comment]
Just want to check about what the process is here, or what boxes need to be checked before this can be merged? Thanks! I'm sure you're all super busy with the Unity refugees so I really appreciate the time (:

**Comment by akien-mga** [comment]
You don't need to do anything further, once approved it's ready to merge by the production team, when we get to it.

**Comment by akien-mga** [comment]
Thanks! And congrats for your first merged Godot contribution :tada:

**Pull request merged**
